pom.xml

```diff
@@ -44,10 +44,10 @@
     <maven.compiler.target>1.7</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
+    <version.org.awaitility-awaitility>3.0.0</version.org.awaitility-awaitility>
     <version.com.squareup.okhttp3-mockwebserver>3.6.0</version.com.squareup.okhttp3-mockwebserver>
     <version.io.github.openfeign>9.4.0</version.io.github.openfeign>
-    <version.io.opentracing>0.22.0</version.io.opentracing>
-    <version.io.opentracing.contrib-opentracing-spanmanager>0.0.5</version.io.opentracing.contrib-opentracing-spanmanager>
+    <version.io.opentracing>0.30.0</version.io.opentracing>
```

Comment:
are there any plans for opentracing to move to semver? seems api breaks are the norm yet the api has been in progress for a while now.